### PR TITLE
Bug 1812719: jsonnet/*: adapt resource requests

### PR DESCRIPTION
**Pull request by s-urbaniak**
This adapts resource requests to lower values.
Rationale: cpu is compressible and we have to make sure we stay within tight boundaries.

Values have been taken from actually used cpu time using the following query: 
```
sum by(pod) (rate(container_cpu_usage_seconds_total{pod=~"POD_NAME*",container=""}[15m])*1024)
```

/cc @openshift/openshift-team-monitoring

### CODE REVIEW

File: assets/alertmanager/alertmanager.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ spec:
       name: web
     resources:
       requests:
-        cpu: 10m
+        cpu: 1m
```
Comment (lilic) [inline assets/alertmanager/alertmanager.yaml]: wow now thats drastic :D

File: assets/prometheus-k8s/prometheus.yaml [hunk 2]
```diff
@@ -154,7 +157,7 @@ spec:
     image: quay.io/thanos/thanos:v0.10.0
     resources:
       requests:
-        cpu: 50m
+        cpu: 1m
```
Comment (lilic) [inline assets/prometheus-k8s/prometheus.yaml]: is this enough for thanos?

Comment (s-urbaniak) [reply assets/prometheus-k8s/prometheus.yaml]: This is for the thanos sidecar next to prometheus. In the e2e run I analyzed ~1millicore CPU:
image

File: assets/alertmanager/alertmanager.yaml (hunk 1 above)
Comment (s-urbaniak) [reply assets/alertmanager/alertmanager.yaml]: this is the alertmanager-proxy (oauth-proxy) sidecar. In the e2e run I analzyed ~1millicore CPU:
image

Comment (smarterclayton) [reply assets/alertmanager/alertmanager.yaml]: Unfortunately anything less than 5m has no effect, the scheduler has an implicit floor at 5m.

Comment (smarterclayton) [reply assets/alertmanager/alertmanager.yaml]: Let me double check this, we have a minimum floor for memory.

Comment (smarterclayton) [reply assets/alertmanager/alertmanager.yaml]: Nm, we don't.  This should be fine.

**Comment by s-urbaniak** [comment]
/hold
master is currently broken in master due to work in user workload monitoring, i will open a PR against 4.4 directly.

**Comment by lilic** [comment]
Can you rebase, this should be ready to go.

**Comment by lilic** [comment]
upgrade test passed but something went wrong with prow:
> 2020/03/13 10:06:25 Build cluster-monitoring-operator succeeded after 1m56s
2020/03/13 10:06:26 Tagging cluster-monitoring-operator into stable

**Comment by lilic** [comment]
/hold cancel

🤷‍♀

**Pull request merged**